docs/en/changes/changes.md

```diff
@@ -52,6 +52,10 @@
 * Deprecated metrics query v2 protocol.
 * Deprecated record query protocol.
 * Add OpenSearch 2.8.0 to test case.
+* Add `ai-pipeline` module.
+* Support HTTP URI formatting through `ai-pipeline` to do pattern recognition.
+* Add new HTTP URI grouping engine with benchmark. 
+* [Breaking Change] Use the new HTTP URI grouping engine to replace the old regex based mechanism.
```

Comment:
FYI @apache/skywalking-committers breaking change in this PR.

Considering a better performance of endpoint grouping, regex-based implementation has been replaced.